torch/quantization/fx/_equalize.py

```diff
@@ -298,6 +322,78 @@ def maybe_get_weight_eq_obs_node(op_node: Node, modules: Dict[str, nn.Module]) -
             return node_arg
     return None
 
+def maybe_get_following_observer(
+    node: Node,
+    modules: Dict[str, nn.Module],
+    target_observer_type: Any = object,
+) -> Optional[Node]:
+    """ Loops through the graph forwards and finds the next observer node with
```

Comment:
if there are multiple paths, looks like this will just follow through one path and find the first observer, might be good to document this I think.